src/xp.runner/Command.cs

```diff
@@ -66,6 +66,22 @@ public int Execute(CommandLine cmd, ConfigSource configuration)
                 ComposerLocations().Select(dir => Paths.Compose(dir, "xp-framework", "core")).Where(Directory.Exists).First()
             };
 
+            Encoding encoding;
+            Func<string, string> args;
+            var main = Paths.TryLocate(configuration.GetUse(), new string[] { Paths.Compose("tools", MainFor(cmd) + ".php") }).FirstOrDefault();
+            if (null == main)
```

Comment:
This is a performance hit, especially for larger USE paths. We could add an INI option reading instead, speeding up this check. On the other hand, this code is exactly what happens today, so it might not be too much of an issue.